src/qt/locale/bitcoin_zh_CN.ts

```diff
@@ -1417,226 +1416,230 @@
     <message>
         <source>Warning: Network Activity Detected</source>
-        <translation type="unfinished"></translation>
+        <translation>警告：网络活动</translation>
     </message>
     <message>
         <source>It is recommended to disconnect from the internet before printing paper wallets. Even though paper wallets are generated on your local computer, it is still possible to unknowingly have malware that transmits your screen to a remote location. It is also recommended to print to a local printer vs a network printer since that network traffic can be monitored. Some advanced printers also store copies of each printed document. Proceed with caution relative to the amount of value you plan to store on each address.</source>
-        <translation type="unfinished"></translation>
+        <translation>建议在打印纸钱包前下线。即使纸钱包在您本地计算机上生成，仍可能有在不知不觉中存在将您的屏幕传输到远程位置的恶意软件。还建议打印到本地而不是网络打印机，因为网络流量可以被监控。某高级打印机还存储每个打印文档的副本。请谨慎前进。</translation>
     </message>
     <message>
         <source>Error encoding Address into QR Code.</source>
-        <translation type="unfinished"></translation>
+        <translation>QR编码地址时出错。</translation>
     </message>
     <message>
         <source>Error encoding private key into QR Code.</source>
-        <translation type="unfinished"></translation>
+        <translation>QR编码密钥时出错。</translation>
     </message>
     <message>
         <source>Printing Error</source>
-        <translation type="unfinished"></translation>
+        <translation>打印出错</translation>
     </message>
     <message>
         <source>failed to open file, is it writable?</source>
-        <translation type="unfinished"></translation>
+        <translation>打开文件失败，你能否写入？</translation>
     </message>
     <message>
         <source>Load Paper Wallets</source>
-        <translation type="unfinished"></translation>
+        <translation>载入纸钱包</translation>
     </message>
     <message>
         <source>The paper wallet printing process has begun.&lt;br/&gt;Please wait for the wallets to print completely and verify that everything printed correctly.&lt;br/&gt;Check for misalignments, ink bleeding, smears, or anything else that could make the private keys unreadable.&lt;br/&gt;Now, enter the number of DOGE you wish to send to each wallet:</source>
-        <translation type="unfinished"></translation>
+        <translation>纸钱包打印已开始。&lt;br/&gt;请等打印完成并验证内容是否打印正确（是否有错位、渗墨、污迹或任何其他可能导致私钥无法读取的情况）。&lt;br/&gt;现在请输入发送到每个钱包的DOGE数量：</translation>
     </message>
     <message>
         <source>Paper wallet %1</source>
-        <translation type="unfinished"></translation>
+        <translation>纸钱包 %1</translation>
     </message>
     <message>
-        <source>&lt;b&gt;%1&lt;/b&gt; to Paper Wallet &lt;span style=&apos;font-family: monospace;&apos;&gt;%2&lt;/span&gt;</source>
-        <translation type="unfinished"></translation>
+        <source>&lt;b&gt;&lt;/b&gt; to Paper Wallet &lt;span style=&apos;font-family: monospace;&apos;&gt;%2&lt;/span&gt;</source>
```

Comment:
changes source - this means this line in the translation will not be applied

Reply:
Apologies, should be fixed now—at least we know that the linter works :)